### PR TITLE
chm-257:  open links in a new window

### PR DESCRIPTION
**Pull request by tahmeedshah**
### General description
Created an AngularJS directive to add the attribute 'target=_blank' to the tags having 'a[href^="https://"]' or 'a[href^="http://"]'.

update km-value-ml to open links in a new window added in forms

### Designs
_Link to the related design prototypes (if applicable)_

### Testing instructions
_Provide minimal instructions on how to test this PR._

* Apart from the added feature / bug fix, check overall performance, styling...

## Checklist before merging
* [ ]  Branch name / PR includes the related Jira ticket Id.
* [ ]  Tests to check core implementation / bug fix added.
* [ ]  All checks in Continuous Integration workflow pass.
* [ ]  Feature functionally tested by reviewer(s).
* [ ]  Code reviewed by reviewer(s).
* [ ]  Documentation updated (README, CHANGELOG...) (if required)

### CODE REVIEW

**Review by sbilodeau — APPROVED** [review]

File: app/components/scbd-angularjs-services/filters/scbd-filters.js [hunk 1]
```diff
@@ -30,7 +30,13 @@ import '../services/locale';
     };		
   }]);
 
-
+  app.filter("targetBlank", function () {
+    return function (value) {
+      const $htmlObject = $(value);
```
Comment (blaisefonseca) [inline app/components/scbd-angularjs-services/filters/scbd-filters.js:35]: 1. No import for $
2. Can you move this code to services and call it `setAnchorTarget(target){}` and pass the target to it. make sure you have `import $ from 'jquery'`

File: app/components/scbd-angularjs-services/filters/scbd-filters.js [hunk 2]
```diff
@@ -222,7 +223,13 @@ import '../services/locale';
     };
   }]);
 
-
+  app.filter("targetBlank", function () {
```
Comment (blaisefonseca) [inline app/components/scbd-angularjs-services/filters/scbd-filters.js:226]: can you use the same name for filter too `setAnchorTarget`

File: app/components/scbd-angularjs-services/services/set-anchor-target.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+export default function setAnchorTarget(value) {
+    const $htmlObject = $(value);
+    $htmlObject.find('a[href^="https://"],a[href^="http://"]').each(function() {
+        if (!$(this).attr('target')) {
+            $(this).attr('target', '_blank');
+        }
+    });
+    return $htmlObject.prop('outerHTML');
+}
```
Comment (blaisefonseca) [inline app/components/scbd-angularjs-services/services/set-anchor-target.js:11]: can you look at `app.directive("translationUrl"`

File: app/components/scbd-angularjs-services/services/set-anchor-target.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+export default function setAnchorTarget(value) {
+
+    var anchorElements = value[0].querySelectorAll('a[href^="https://"], a[href^="http://"]');
+
+    anchorElements.forEach(function (anchorElement) {
+        if (anchorElement && !anchorElement.getAttribute('target')) {
+            anchorElement.setAttribute('target', '_blank');
```
Comment (blaisefonseca) [inline app/components/scbd-angularjs-services/services/set-anchor-target.js:7]: can you make the target parameter

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**